roles/maps/tasks/download_large_file.yml

```diff
@@ -56,26 +69,39 @@
           size_disp = str(round(size / MiB, 2)) + ' MiB'
         else:
           size_disp = str(round(size / GiB, 2)) + ' GiB'
-        print(' (' + size_disp + ') To check progress, run: "tail {{ working_dir }}/{{ log_file }}"')
+        print('(' + size_disp + ') To check download progress see above.')
       args:
         executable: /usr/bin/python3
-      vars:
-        # Where the file eventually goes
-        dest_path: "{{ dest_base_path }}/{{ item }}"
 
-        log_file: "{{ item }}.log"
-        working_dir: "/library/downloads/maps"
-      register: download_file_size
+      # The existence of console output in download_file_details
+      # implies that the download should happen.
+      register: download_file_details
+
+    # Put in some space to make it easier to read by implementers.
+    - debug:
+        msg:
+          - ""
+          - ""
+          - "   To check download progress, run:       "
```

Comment:
I think I've tried this before but multiline debug messages don't work right?

    msg: |
         
         like
         this

kinda annoying that doesn't work